### DISCUSSION TITLE
Question about accessing the manifest.json file

### DISCUSSION THREAD

**Discussion by CGurg** [learningeconomy/LearnCard#299] [Help] [answered]
Hello everyone, I have been working on trying to use LearnCard with our site. We have recently switched to using the Remix framework for our site and I am having a bit of an issue with LearnCard being unable to access the manifest.json file I added to my project due to a CORS error. 

I am relatively new to using Remix so I have been looking for a solution to allow LearnCard to access that file. I appreciate any help or input that anybody can provide.

**Comment by Custard7** [comment]
Hey @CGurg - great to hear you are integrating LearnCard!

Are you able to post additional information about the error - which endpoints are you receiving CORS errors from?

Are you attempting to setup a CHAPI manifest file?  

Cheers,
Jacksón

**Reply by CGurg** [reply]
Yes, I am attempting to set up a CHAPI manifest file. I have pretty much been following this guide: https://docs.learncard.com/learn-card-sdk/learncard-core/chapi/chapi-wallet-setup-guide

The CORS error in the console reads "Access to fetch at 'http://localhost:3000/manifest.json' from origin 'https://authn.io' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled."

I have been looking for a way to allow our manifest file to be retrieved, but have been struggling to find a solution. I wasn't sure if there was a way to pass the relevant information without the manifest file or another solution I haven't thought of.

Thank you!

**Reply by dmitrizagidulin** [reply, marked as answer]
Hi @CGurg. I'm not familiar with Remix framework unfortunately. But in Javascript, you're essentially looking to enable CORS headers on the route where you're hosting your static `manifest.json` file. I suspect you can use something like https://www.npmjs.com/package/cors (adding the `cors()` handler to the route) with Remix.

**Reply by dmitrizagidulin** [reply]
> I wasn't sure if there was a way to pass the relevant information without the manifest file

It's required, yeah, no real good way around it but to enable the headers.

**Reply by CGurg** [reply]
@dmitrizagidulin Thank you! I will work on setting that up and post back here if I get it working.
